src/python-imports-search.ts

```diff
@@ -52,6 +52,10 @@ export function activate(
                     return
                 }
 
+                if (codeEditor.type !== 'CodeEditor') {
+                    return
+                }
+
```

Comment:
I guess, we could combine all early return statements + this will also unnest one level:
```js
if (
    !settings['pyImports.showAllUsagesLinks'] ||
    codeEditor.type !== 'CodeEditor' ||
    codeEditor.document?.languageId !== 'python' ||
    !codeEditor.document.text
) {
    return
}

const text = codeEditor.document.text
// ...
```
But it is up to you as always!